src/doc/contrib/src/process/unstable.md

```diff
@@ -42,6 +42,32 @@ issues are filed, they can be easily tied together.
 [tracking issue]: https://github.com/rust-lang/cargo/labels/C-tracking-issue
 [tracking issue template]: https://github.com/rust-lang/cargo/issues/new?labels=C-tracking-issue&template=tracking_issue.md
 
+## Pre-Stabilization 
+
+Once an unstable feature is "complete", the search for users to test
+and give feedback begins. Testing notes should be written up to give users an
+idea of how to test the new feature. An example being the 
+[workspace inheritance testing notes] for workspace inheritance. Once testing
+notes have been written up you should make posts in various rust communities
+([rust subreddit], [users], [internals], etc). Example posts made for workspace
+inheritance: [reddit post], [users post], [internals post]. The unstable feature
+should also be added to [This Week in Rust]. This should be done by adding the
+label `call-for-testing` to the RFC for the feature and making a comment with a
```

Comment:
Is this label a thing now? Someone already created it in rust-lang/rfcs, but the discussion of its mechanism seems not yet finished as far as I see. 

Apart from that, the write-up looks good to me :)

Reply:
Fair enough. Thanks for the clarification! I am going to merge this, and we can always tweak it afterwards.

Reply:
The issue that was tracking how `call-for-testing` should work, got closed so it's a bit more set in stone. I'll keep and eye on any future updates of it and update the docs as needed. Thanks for merging this!